JavaScript/JavaScriptSDK/TelemetryContext.ts

```diff
@@ -65,10 +67,22 @@ module Microsoft.ApplicationInsights {
         public session: Context.Session;
 
         /**
+        * Custom properties added to all telemetry items (implementation of the TelemetryInitializer concept).
+        */
+        public properties;
+
+        /**
+        * Custom measurements added to all telemetry items (implementation of the TelemetryInitializer concept).
+        */
+        public measurements;
+        
+        public onInitializeTelemetry: (envelope: Telemetry.Common.Envelope) => boolean;
+
```

Comment:
There are several options what to pass to the callback - all of them narrow the possibilities comparing to passing the whole envelope. And even worse - they make me guessing which one is more useful/needed more. I don't want to guess, so passing the whole thing and waiting for customers' feedback. Does that sound good?